"what's the point of the return in this code? so, i have this function below:..def remove_all(lst):.    i = 0.    while i &lt; 10:.        try:.            print('removing... ').            print(int(lst.pop()) + 10).            print(""removed successfully."")..        # as soon as an indexerror is raised, jump to the following block of code...        .        except indexerror as err: .            # if you encounter an indexerror, do the following:.            print(""uh oh! problems."").            return..        #as soon as a value error is raised, jump here..        except valueerror as err:.            print(""not a number"")..        i = i + 1...what does the return do? there is no value after the return, so does it mean none or true?.and what is the point of having the return there if the value is none?..thanks!"
